github-runner-manager/src/github_runner_manager/configuration/base.py

```diff
@@ -14,6 +15,21 @@
 logger = logging.getLogger(__name__)
 
 
+# The github-runner-manager is being refactor from a library to an application.
+# Once the charm no longer rely on the github-runner-manager as a library this will be removed.
```

Comment:
can you add the rationale why this will be removed

Reply:
Added.